packages/wallets/src/features/cashier/modules/ResetBalance/ResetBalanceContent.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { DerivLightDemoResetBalanceIcon, DerivLightDemoResetBalanceSuccessfulIcon } from '@deriv/quill-icons';
+import { Localize, useTranslations } from '@deriv-com/translations';
+import { ActionScreen, Button } from '@deriv-com/ui';
+
+export const getResetBalanceContent = (
+    localize: ReturnType<typeof useTranslations>['localize'],
+    resetBalance: () => void,
+    navigateToTransfer: () => void
+) => ({
```

Comment:
@lubega-deriv, this seem like some what a utility function. Does it belong in the `utils` directory or something? 
Also, the name of the file suggests it's a component. 🤔

Reply:
hmm I don't think this should be moved into the utils folder as it's a mapper specific to reset balance. I'll update the structure ot avoid confusion 🙏